client/extensions/woocommerce/state/wc-api/shipping-zones/selectors.js

```diff
@@ -9,7 +9,13 @@ import { get, isArray } from 'lodash';
 import { getSelectedSiteId } from 'state/ui/selectors';
 import { LOADING } from './reducer';
 
-const getRawShippingZones = ( state, siteId ) => {
+/**
+ * @param {Object} state Whole Redux state tree
+ * @param {Number} [siteId] Site ID to check. If not provided, the Site ID selected in the UI will be used
+ * @return {Object} The list of shipping zones, as retrieved fro the server. It can also be  the string "LOADING"
+ * if the zones are currently being fetched, or a "falsy" value if that haven't been fetched at all.
+ */
+export const getAPIShippingZones = ( state, siteId = getSelectedSiteId( state ) ) => {
```

Comment:
Optional: I think API here is redundant since it's under the `wc-api` directory already.